server.js

```diff
@@ -8,7 +8,7 @@ if (!process.env.DEMO_DEPLOY) {
 }
 
 const server = require('./app');
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || 5000;
```

Comment:
i made `npm start ` do both the client and the server (although we lose the nice interactive console that `react-scripts start` gives us - is that better?